src/screens/users/index.js

```diff
@@ -1,78 +1,90 @@
 import React, { PureComponent } from 'react';
-import { View, Text, TouchableOpacity } from '../../components';
-import { StyleSheet } from 'react-native'
+import { StyleSheet, TextInput } from 'react-native';
 import { connect } from 'react-redux';
+import { Text, View, Touchable } from '../../components';
+import * as COLORS from '../../constants/colors';
+import { search } from '../../store/users/actions'
 
 class Users extends PureComponent {
-    constructor() {
-        super();
-    }
-
     openChat = (receivingUser) => {
         this.props.navigation.navigate('Chat', { receivingUser });
     }
 
+    handleChange = (type, value) => {
+        this.setState({ [type]: value });
+    }
+
+    handleSubmit = () => {
```

Comment:
I think it's more beautiful, but you use 5 additional lines for one-liner.
```
  handleSubmit = () => {
        const { name } = this.state
        const { onSearch } = this.props

        onSearch(name)
    }
```
I think it's better way. This is also understandable as 5 line code.
```
  handleSubmit = () => this.props.onSearch(this.state.name)
```

Reply:
This is inconvenient for expanding the functionality, especially at the development stage.